episodes/troubleshooting.md

```diff
@@ -83,15 +141,24 @@ Considering your responses to the prompts above, identify which of the following
 - documentation
 - training
 
-Is there anything else you need, to support hub users?
+Is there anything else you need to support hub users?
 
 ::::
 
+## Wrap-up
+
+Thank you for following the Hub Champion Training! 🏅
+
+We hope you found it useful for building your skills and confidence as a Hub Champion to empower your community to make best use out of the hub service.
+
+We're always looking to improve our lesson and welcome your feedback.
+
 :::{card} 
 KEY POINTS
 ^^^
-After following this episode, learners will be able to...
+After following this episode, Hub Champions will be able to...
+
+- Choose the best place to look or ask for help with hub management
+- Formulate a plan to support their community
```

Comment:
Ooh okay I am consulting the excellent Collaborative Lesson Development Training materials:

> a [completed episode also requires](https://carpentries.github.io/sandpaper-docs/episodes.html?#required-elements) fenced divs for the questions the episode answers and the keypoints a lesson covers.

I suggest:

```suggestion
- Part of your role as Hub Champion is to adapt and respond to the specific needs of your community
- Consult [further resources](ref:resources) may help you troubleshoot some common issues
- If you are stuck and require technical help on behalf of your users, then you can contact 2i2c through their support desk
- Make a plan for your support process to set yourself up for success as a Hub Champion
- Formulate a plan to support their community
```

Reply:
This looks great, thanks